Makefile

```diff
@@ -71,7 +71,7 @@ generate-go: mockgen
 	go generate ./...
 
 .PHONY: verify-generated
-verify-generated: generate
+verify-generated: generate generate-openshift
```

Comment:
FYI, you could move this dependency into openshift.mk, too. Put the following in openshift.mk:

```
verify-generated: generate-openshift
```

That just defines an additional dependency without overriding the rule.

Reply:
TODAY I LEARNT